### DISCUSSION TITLE
about gItf/glb compression

### DISCUSSION THREAD

**Discussion by Yan-Xiaodi** [google/filament#4123] [Q&A]
Recently I want to compress the gltf/glb file ,and I konw  gltf-pipeline  can do it,but it rely on the Node.js；Dose Filament has the similar Library to  help the gltf compression? Or any other way to do it ,I have noticed Draco only supported .obj or .ply file. My problem may seemed  a little stupid,I'm green hand on this.Can someone help me? Thanks

**Comment by romainguy** [comment]
Filament does not provide a glb/gltf compressor. We do support compressed glTF files though. Besides gltf-pipeline you can use Blender for instance.

**Reply by Yan-Xiaodi** [reply]
Thanks for your quick reply，it help me a lot. I will follow your proposal to see the Blender。Thank you
